cpp/tests/utilities/column_utilities.cu

```diff
@@ -375,6 +375,41 @@ static auto numeric_to_string_precise(T value)
   return o.str();
 }
 
+template <typename T,
+          typename std::enable_if_t<std::is_same<T, cudf::duration_D>::value>* = nullptr>
+static auto duration_suffix(T)
+{
+  return " days";
+}
+
+template <typename T,
+          typename std::enable_if_t<std::is_same<T, cudf::duration_s>::value>* = nullptr>
+static auto duration_suffix(T)
+{
+  return " seconds";
+}
+
+template <typename T,
+          typename std::enable_if_t<std::is_same<T, cudf::duration_ms>::value>* = nullptr>
+static auto duration_suffix(T)
+{
+  return " milliseconds";
+}
+
+template <typename T,
+          typename std::enable_if_t<std::is_same<T, cudf::duration_us>::value>* = nullptr>
+static auto duration_suffix(T)
+{
+  return " microseconds";
+}
+
+template <typename T,
+          typename std::enable_if_t<std::is_same<T, cudf::duration_ns>::value>* = nullptr>
+static auto duration_suffix(T)
```

Comment:
No need to use enable_if here. Just use normal function overloading. Much cleaner:
```
auto duration_suffix(cudf::duration_ns){
return " nanoseconds";
}
```

Reply:
done